### PR TITLE
Local startup

### PR DESCRIPTION
**Pull request by chemik-bit**
Added option for local startup

### CODE REVIEW

File: api-website/conf.py [hunk 1]
```diff
@@ -28,10 +28,42 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
+<<<<<<< HEAD
```
Comment (filip-michalsky) [inline api-website/conf.py:31]: @chemik-bit  there is some merge conflict here?

File: api-website/_build/html/salesgpt/salesgpt.templates.html [hunk 2]
```diff
@@ -1,11 +1,20 @@
 <!DOCTYPE html>
 <html class="writer-html5" lang="en" data-content_root="../">
 <head>
-  <meta charset="utf-8" /><meta name="generator" content="Docutils 0.18.1: http://docutils.sourceforge.net/" />
-
+  <meta charset="utf-8" /><meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <!-- Google tag (gtag.js) -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-N73Z2D8W81"></script>
```
Comment (filip-michalsky) [inline api-website/_build/html/salesgpt/salesgpt.templates.html:7]: Is it an issue to expose our gtag token

File: frontend/start-dev.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+export $(grep -v '^#' ../.env | xargs)
```
Comment (filip-michalsky) [inline frontend/start-dev.sh:2]: did we add to documentation that frontend env vars are managed from top dir?

File: run_api.py [hunk 4]
```diff
@@ -77,7 +77,7 @@ async def get_bot_name(authorization: str = Header(...)):
 
 
 @app.post("/chat")
-async def chat_with_sales_agent(req: MessageList, stream: bool = Query(False), authorization: str = Header(...)):
+async def chat_with_sales_agent(req: MessageList, stream: bool = Query(False), authorization: Optional[str] = Header(None)):
```
Comment (filip-michalsky) [inline run_api.py:80]: nice!

**Review by filip-michalsky — APPROVED** [review]

File: api-website/conf.py (hunk 1 above)
Comment (chemik-bit) [reply api-website/conf.py:31]: Yep, an old version was still present, fixed :)

File: frontend/start-dev.sh (hunk 3 above)
Comment (chemik-bit) [reply frontend/start-dev.sh:2]: Not yet, will be in next PR along with docstrings and other updates

**Pull request merged**